README.rst

```diff
@@ -1,6 +1,6 @@
-==========================================
+============
 SLSim (Strong Lensing Simulation) pipeline
-==========================================
+============
```

Comment:
You need the '===' on the same length as the title otherwise formatting will not work well

Reply:
Thanks Simon! I realized this and already fixed them.